### DISCUSSION TITLE
Use case for additonal_information - APC pricing

### DISCUSSION THREAD

**Discussion by PricedTransparently** [CMSgov/price-transparency-guide#440] [General]
Howdy, my organization has recently implemented the additional_information object for a specific pricing scenario, and the purpose of this post is to hopefully gain some feedback on whether or not this is an appropriate use of the additional_information object. 

The pricing scenario in question applies to APC codes, specifically when they are priced using a third party pricer. In this scenario, rates will sometimes be calculated as a percentage of some external price (which we cannot retrieve in a price transparency context). This means our negotiated_type does not fit neatly into the "percent" or "negotiated" box, seeing as the percent reported is not a percent of billed, nor is the percentage being applied to a contracted rate. Our solution for handling this scenario is to report the percentage as the negotiated_rate, with a negotiated_type of "percent". Additionally, we will include an additional_information object which states, "This rate is calculated as the percentage of an external price."

 Any feedback on this approach would be greatly appreciated, especially if there are any concerns with using the additional_information object in this way. Thanks!

**Comment by jltang-kp** [comment]
Our plan is similar for any pricing scenario that will use a PPS pricer it use the percentage of the pricer and have comments in the additional info field. Since we will not be able use the PPS pricer to "call" the base rate the best we can do is also storing that percent modifier. 

Our vendor that is supporting our claims system is assisting us on developing the MRF output and agrees that somehow calling the PPS pricer would not be possible in this context as well. Just curious is your system inhouse for generating the MRF or are you using a vendor as well?

**Comment by shaselton-usds** [comment]
@PricedTransparently -- Thanks for the heads up. Just to make sure I understand correctly, the difficulty is obtaining the base-rate for the third-party party pricers for APC codes, yes? Are the third-party pricers based on some other reference such as Medicare? Just trying to understand how a negotiation would take place that would be percentage-based without actually knowing the reference or base-rate. This should be a "negotiated" situation rather than "percentage".

@jltang-kp -- For pricing scenarios that leverage PPS pricers, would be able to walk me through why a base rate would be impossible to get at?

**Reply by joseph-kendall** [reply]
Can you please provide more feedback? I imagine many are going to have this same situation. We have the same situation where we need to run a claim through the pricer to get the allowed amount. Without the claim, all we know is a percentage in the contract of the pricer that we pay.

**Reply by RLTx1391** [reply]
@shaselton-usds any feedback on this scenario?

**Comment by jltang-kp** [comment]
@shaselton-usds So just to clarify as well the pricer vendor we use handles both APCs and DRGs. Also our pricing vendor is not the same vendor that is working on the producing the in-network-file from our claims system. 

So some of our contracts have a subset of billing codes that would be routed to a pricer for pricing. Usually to get a rate to be returned from the pricer we use we need claim specific info for the pricer to price, such as patient type, patient status, gender, age, type of bill. This would normally work fine if we had a claim to price.

In discussions with our pricing vendor they would be able to provide a "mock claim" file with representative patients, not in the schema format but in hopes that we could ingest that file to run against our contracts that use a pricer to produce the rates needed in-network-file. Our vendor that is producing the in-network-file indicated that they would not be able to ingest that "mock claim" file so the closest we can get is that percentage modifier.

**Comment by CKreklow** [comment]
From the perspective of a transparency consumer, it would seem that the underlying criteria driving the price variants should be reported.  Otherwise there's a rather large loophole of outsourcing pricing to a third-party and then saying "we don't know how they generate rates".

**Reply by jwerner15083** [reply]
Actually it is not a loophole at all.  CMS based contracting (% of Medicare) is commonly used.  Understanding though the values specific to garnering the final value of the claim itself, and then unpacking that into a MRF line, is simply not always possible.  OPPS for example,, CMS has never been able to put out their own PC pricing tool due to the complexity of that "formula", and now we are being asked to display (for a specific provider) all of the potential rates?  APC bundling is not that simple as it is a formula, and most companies rely on third party vendors such as 3M or Microdyn to provide that service.  So determining the underlying "value" to create a table of payment values is simply not possible in all cases.

@shaselton-usds This remains a common question that does not seem to be answered.  If you have a simple contract based on a % of CMS derived system (lets say OPPS), can we display this with a negotiated_type = "OPPS" and the negotiated_rate = the % used?  And filling in the additional_information with something such as "Priced at X% of OPPS", etc.

**Comment by ayalliant** [comment]
@shaselton-usds - I apologize if I have missed it but has there been a response to the question regarding how we should report contracts that are a "Percent of CMS"?
